pkg/kubelet/apis/config/types.go

```diff
@@ -67,6 +67,12 @@ const (
 	// SingleNumaNodeTopologyManager Policy iis a mode in which kubelet only allows
 	// pods with a single NUMA alignment of CPU and device resources.
 	SingleNumaNodeTopologyManager = "single-numa-node"
+	// ContainerTopologyScope represents that
+	// topology policy is applied on a per-container basis.
+	ContainerScopeTopology = "container"
```

Comment:
I think `ContainerTopologyScope` as in the comment above is a better name. As minimum they should match

Reply:
Done, changed:
`containerScopeTopology` -> `containerTopologyScope`
`podScopeTopology` -> `podTopologyScope`